app/views/katello/api/v2/content_facet/base.json.rabl

```diff
@@ -0,0 +1,11 @@
+
+attributes :content_view_id, :content_view_name
+attributes :lifecycle_environment_id, :lifecycle_environment_name
+attributes :uuid
+
+node :errata_counts do |content_facet|
+  if content_facet.host.content_host
```

Comment:
I think showing the difference between "we calculated the errata and these are the results" and "we don't have this information" is important.